website/react-magma-docs/src/pages/api/select.mdx

```diff
@@ -159,6 +159,71 @@ export function Example() {
 }
 ```
 
+## Disabled Items
+
+To disable specific items in the `Select` component, add a `disabled: true` property to each item object. Disabled items specified as `defaultSelectedItem` or `initialSelectedItem` will not be selected by default or initially. If `initialHighlightedIndex` points to a `disabled` item, the component will not highlight it.
+
+Example:
+
+```tsx
+import React from 'react';
+import { Select } from 'react-magma-dom';
+
+export function Example() {
+  return (
+    <Select
+      id="disabledItemsSelectId"
+      labelText="Select with Disabled Items"
+      items={[
+        { label: 'Red', value: 'red' },
+        { label: 'Blue', value: 'blue', disabled: true },
+        { label: 'Green', value: 'green' },
+      ]}
+      isClearable
+      initialHighlightedIndex={1}
+      defaultSelectedItem={{ label: 'Blue', value: 'blue', disabled: true }}
+      initialSelectedItem={{ label: 'Blue', value: 'blue', disabled: true }}
+    />
+  );
+}
+```
+
+
+```tsx
```

Comment:
Instead of 2 separate sandboxes/jsx blocks for the examples, we usually just have 1 with both components we want to showcase.

Reply:
fixed